unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/NftTypePreviewCameraFocusSO.cs

```diff
@@ -0,0 +1,26 @@
+﻿using MainScripts.DCL.Controllers.HUD.CharacterPreview;
+using System;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "NftTypePreviewCameraFocus", menuName = "Variables/NftTypePreviewCameraFocus")]
+public class NftTypePreviewCameraFocusSO : ScriptableObject
```

Comment:
I'd call it `NftTypePreviewCameraFocusConfig`.

Also I dont think we need this as a project-wide condig. It is something super specific to the BackpackV2View, it can be serialized there in the monobehaviour.

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/5162/commits/13cc4d2ccb30e3422d397bcf710401c910318f53